src/components/LockButton.jsx

```diff
@@ -13,6 +13,7 @@ const LockButton = ({onClick, readOnly}) => {
       label={`Switch to ${ readOnly ? 'edit' : 'read-only' } mode`}
       onTouchTap={onClick}
       tooltip={toolTipText}
+      style={{zIndex: 5}}
```

Comment:
I mentioned this on one of Jordan's PRs as well: I think we should avoid inline styling in this project, and use CSS files/classes instead.